tests/ZfcRbacTest/Factory/AuthorizationServiceDelegatorTest.php

```diff
@@ -119,10 +154,17 @@ function () {
             'ZfcRbac\Factory\AuthorizationServiceDelegatorFactory'
         );
 
-        $this->setExpectedException(
-            'ZfcRbac\Exception\RuntimeException',
-            'The service ZfcRbacTest\AuthorizationAware must implement AuthorizationServiceAwareInterface.'
-        );
-        $serviceManager->get('ZfcRbacTest\AuthorizationAware');
+        $thrown = false;
+        try {
+            $serviceManager->get('ZfcRbacTest\AuthorizationAware');
+        } catch (\Exception $e) {
```

Comment:
the `ZfcRbac\Exception\RuntimeException` is not thrown, it's only the  previous exception in zf3, because a ServiceNotCreatedException is thrown in zf3.